olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -232,49 +232,82 @@ spec:
           selector:
             matchLabels:
               name: knative-serving-operator
-          strategy: {}
           template:
             metadata:
+              annotations:
+                sidecar.istio.io/inject: "false"
               labels:
                 name: knative-serving-operator
             spec:
               containers:
-              - command:
-                - knative-serving-operator
-                env:
-                - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
-                - name: NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.namespace
+              - env:
                 - name: POD_NAME
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: knative-serving-operator
-                - name: IMAGE_QUEUE
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-queue
-                - name: IMAGE_activator
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-activator
-                - name: IMAGE_autoscaler
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-autoscaler
-                - name: IMAGE_autoscaler-hpa
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-autoscaler-hpa
-                - name: IMAGE_controller
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-controller
-                - name: IMAGE_networking-istio
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-istio
-                - name: IMAGE_webhook
-                  value: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-webhook
-                image: $IMAGE_KNATIVE_SERVING_OPERATOR
-                imagePullPolicy: Always
+                - name: SYSTEM_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: METRICS_DOMAIN
+                  value: knative.dev/serving-operator
+                image: gcr.io/knative-releases/knative.dev/serving-operator/cmd/manager:v0.11.1
```

Comment:
We'll need to build our own image of this rather than straight up use upstream's. Let's discuss how we can do that.